spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/backendService/SCSpaconBackend.java

```diff
@@ -52,15 +54,14 @@
 import rx.subjects.BehaviorSubject;
 
 import static com.boundlessgeo.spatialconnect.scutilities.Json.SCObjectMapper.getMapper;
-import static java.util.Arrays.asList;
 
 /**
- * SCBackendService handles any communication with backend SpatialConnect services.
+ * SCSpaconBackend handles any communication with backend services through an MQTT broker.
  */
-public class SCBackendService extends SCService implements SCServiceLifecycle {
+public class SCSpaconBackend implements ISCBackend {
 
-    private static final String LOG_TAG = SCBackendService.class.getSimpleName();
-    private static final String SERVICE_NAME = "SC_BACKEND_SERVICE";
+    private static final String LOG_TAG = SCSpaconBackend.class.getSimpleName();
+    private static final String SERVICE_NAME = "SC_SPACON_BACKEND_SERVICE";
```

Comment:
We probably can remove the service name, this was the old approach instead of using a backendType, its technically not a service in the sense of the other services

Reply:
Good catch...I'll remove it